src/ring-call-channel.c

```diff
@@ -544,6 +553,15 @@ ring_call_channel_class_init(RingCallChannelClass *klass)
       G_PARAM_READABLE | G_PARAM_STATIC_STRINGS));
 
   g_object_class_install_property(
+    object_class, PROP_IMSI,
+    g_param_spec_string("imsi",
+      "IMSI",
+      "Internation Mobile Subscriber Identifer",
```

Comment:
It is Identity here https://github.com/nemomobile/telepathy-ring/blob/9a51da9a87309542c12d6b9674b88d0369543e17/modem/sim-service.c#L282